examples/http_crawl.cpp

```diff
@@ -49,7 +49,7 @@ int main(int, char const*[])
             beast::http::read(sock, sb, res);
             std::cout << res;
         }
-        catch(boost::system::system_error const& ec)
+        catch(beast::system_error const& ec)
```

Comment:
The reason that beast defines these aliases is to support namespace injection. i.e. choose between `std::error_code` and `boost::system::error_code` at compile time.